src/cmd/serve.rs

```diff
@@ -631,7 +637,7 @@ fn detect_change_kind(pwd: &Path, path: &Path) -> (ChangeKind, PathBuf) {
         ChangeKind::StaticFiles
     } else if partial_path.starts_with("/sass") {
         ChangeKind::Sass
-    } else if partial_path == Path::new("/config.toml") {
+    } else if partial_path.extension().unwrap() == "toml" {
```

Comment:
I'm not entirely sure about this one. Basically any TOML file is treated as Zola's `config.toml`. Should we somehow enforce both "config" and "toml" to be present?

Reply:
You can pass the config filename to the function so it watches the correct one

Reply:
@Keats good point :+1: Fixed!

Reply:
Fixing tests...